client/src/components/Onboarding/OnboardingStatus.module.scss

```diff
@@ -3,9 +3,8 @@
   margin-right: 0.5em;
 }
 
-.loading-icon {
-  display: inherit;
```

Comment:
This may be what fixes the jiggle on loading… Wes, try restoring it and testing for jiggle (which has returned).

Reply:
Problem fixed. Wasn't this.